documentation/Overview.md

```diff
@@ -0,0 +1,88 @@
+<!--
+Copyright (c) 2019, NVIDIA CORPORATION.  All rights reserved.
+-->
+
+# Overview of Compiler Phases
+
+Each phase produces correct output or fatal errors.
```

Comment:
It's a nit, but most of the "Input:" descriptions have a "." at the end while two of them don't.